frontend/src/lib/components/chart/ChartHomeBlock.svelte

```diff
@@ -31,26 +31,28 @@
 	};
 </script>
 
-<button
-	class="flex flex-col items-center border-2 border-grey-lighter rounded-lg m-2 hover:bg-primary-light w-48 h-24"
+<div
+	class="flex flex-col items-center border-2 border-grey-lighter rounded-lg hover:bg-primary-ligther mt-2 cursor-pointer p-2 mr-2 max-w-[500px]"
 	on:click={() => goto(`${$page.url}/${chart.id}`)}
+	on:keypress={() => goto(`${$page.url}/${chart.id}`)}
```

Comment:
Why make this a `div` when it acts as a button? I think this kinda destroys accessibility, no?